config/config.go

```diff
@@ -872,10 +872,12 @@ const (
 	PackageTypeModule PackageType = "module"
 	// PackageTypeSlamMap represents a slam internal state.
 	PackageTypeSlamMap PackageType = "slam_map"
+	// PackageTypeBoardDefs represents a linux board definition file.
+	PackageTypeBoardDefs PackageType = "board_defs"
 )
 
 // SupportedPackageTypes is a list of all of the valid package types.
-var SupportedPackageTypes = []PackageType{PackageTypeMlModel, PackageTypeModule, PackageTypeSlamMap}
+var SupportedPackageTypes = []PackageType{PackageTypeMlModel, PackageTypeModule, PackageTypeSlamMap, PackageTypeBoardDefs}
```

Comment:
Suggest making it singular `PackageTypeBoardDef` instead of plural `PackageTypeBoardDefs`to match the rest. But I think the API work is already done.

Reply:
I'm open to disagreement, but I like the S at the end. These packages could contain multiple files, one for each configuration of a certain board (e.g., one for the factory-reset version, and another for when you've enabled the PWM overlays).

Reply:
Ah! ok then :)